### DISCUSSION TITLE
Other meteorological data to ARL

### DISCUSSION THREAD

**Discussion by elenafillo** [uataq/stilt#127] [Meteorology]
Hi,
I'm struggling to convert ERA5 met data to ARL format. I have tried grib2arl (https://www.ready.noaa.gov/HYSPLIT_data2arl.php) and era52arl (https://github.com/amcz/hysplit_metdata), but neither of them have worked for me. Are you planning to offer support on converting ERA5 files to ARL, or are there any converters you recommend?

**Comment by benfasoli** [comment]
I haven't used ERA5 but since STILT is a superset of HYSPLIT tools, most meteorology that is compatible with HYSPLIT can be used with STILT. Beyond that, converting data to/from HYSPLIT's ARL format isn't really within STILT's scope. The easiest approach is likely the `era52arl` conversion utility maintained by NOAA ARL, and the [HYSPLIT community forum](https://hysplitbbs.arl.noaa.gov/viewtopic.php?t=1351) would be a good place to get support.

NOAA ARL makes other meteorology products available (e.g. GFS and HRRR) that you can use out of the box, which might be easier than fighting with the converters.

**Comment by abdelazizmotiaa** [comment]
Hello Mr. @benfasoli,
I have encountred a problem working on the XSTILT, runing the code compute_bg.r 
and I would appreciate your help, in fact I have configured my receptor that I want, and it runs without generate the file PARTICLES.DAT
the reason it gives in the .out file is that it is unable to locate the 20160605_gfs file, akthough I am sure that the file is there(I also runned using it the backward xstilt.r code)
I appreciate you help if you can tell me how to solve this problem
attached the error and control and slurm.out file
[compute_bg.zip](https://github.com/user-attachments/files/23748003/compute_bg.zip)
[file.zip.zip](https://github.com/user-attachments/files/23748004/file.zip.zip)
Thank you
